cli/azd/extensions/azure.ai.agents/internal/cmd/init.go

```diff
@@ -1415,5 +1416,23 @@ func (a *InitAction) getModelDetails(ctx context.Context, modelName string, mode
 		return nil, fmt.Errorf("failed to get model deployment: %w", err)
 	}
 
+	if modelDeployment.Sku.Capacity == -1 {
+		skuCapacity, err := a.azdClient.Prompt().Prompt(ctx, &azdext.PromptRequest{
+			Options: &azdext.PromptOptions{
+				Message:        "Selected model SKU has no default capacity. Please enter desired capacity",
+				IgnoreHintKeys: true,
+			},
```

Comment:
To make sure the user enters a value

```suggestion
				Message:        "Selected model SKU has no default capacity. Please enter desired capacity",
				IgnoreHintKeys: true,
				Required:       true,
			},
```